templates/CRM/Pledge/Form/Pledge.tpl

```diff
@@ -51,9 +51,9 @@
             <td>{$form.installments.html} {ts}installments of{/ts}
               <span class='currency-symbol'>
                 {if $action eq 1 or $isPending}
-                  {$form.eachPaymentAmount.html|crmMoney:$currency}
+                  {$form.eachPaymentAmount.html}
                 {elseif $action eq 2 and !$isPending}
-                  {$eachPaymentAmount|crmMoney:$currency}
+                  {$eachPaymentAmount}
```

Comment:
The first one is ok, just the same dollar sign thing.

This second one that isn't a form element, it _needs the crmMoney_ otherwise it's both missing the dollar sign and in another locale it doesn't reformat the separators properly.